.github/workflows/upstream.yml

```diff
@@ -122,7 +132,7 @@ jobs:
           -v $GITHUB_OUTPUT:/fatmouse/output \
           -v ~/.scalr-labs:/etc/scalr-labs \
           fatmouse/python-builder:master python -u clickfile.py te up \
-          ${FATMOUSE_BRANCH} ${SCALR_BRANCH} ${IMAGE} \
+          ${FATMOUSE_BRANCH} ${IMAGE} \
```

Comment:
for context - `cli te up` command will check exactly same branch presence in int-scalr, so this input is redundant.